node/network/statement-distribution/src/lib.rs

```diff
@@ -624,9 +624,9 @@ struct ActiveHeadData {
 	statements: IndexMap<StoredStatementComparator, SignedFullStatement>,
 	/// Large statements we are waiting for with associated meta data.
 	waiting_large_statements: HashMap<CandidateHash, LargeStatementStatus>,
-	/// The validators at this head.
+	/// The parachain validators at the head's child session index.
 	validators: Vec<ValidatorId>,
-	/// The session index this head is at.
+	/// The session index this head's child is at.
```

Comment:
```suggestion
	/// The session index this head's child will be at.
```
I think

Reply:
Should I call this the current session in the context of a specific fork ?

Reply:
I am not 100% sure about the wording here myself.

Reply:
@eskimor @ordian would be great if you could chime in here 😄